support/node-production-aws-sns/index.js

```diff
@@ -0,0 +1,62 @@
+const { ApiPromise, WsProvider } = require('@polkadot/api');
+const AWS = require('aws-sdk');
+const nodeCron = require('node-cron');
+const settings = require('./settings.json')
+
+// Create a promise API instance of the passed in node address.
+async function createPromiseApi(nodeAddress, types) {
+    const provider = new WsProvider(nodeAddress);
+    const api = await new ApiPromise({ provider, types });
+    await api.isReady;
+    return api;
+}
+
+async function publishToTopic(topicName, message) {
+    AWS.config.update({region: settings.region});
+    const sns = new AWS.SNS({apiVersion: settings.awsApiVersion});
+    const topics = await sns.listTopics().promise();
```

Comment:
How long does this RPC take? If it's long, we may want to do this when the script starts to avoid additional latency when detecting problems.